packages/block-editor/src/components/preview-options/style.scss

```diff
@@ -40,29 +39,12 @@
 	}
 }
 
-.editor-post-preview__grouping-external {
-	display: flex;
-	position: relative;
-}
-
-.editor-post-preview__button-external {
-	padding-left: $grid-unit-10;
-	margin-right: auto;
-	width: 100%;
-	display: flex;
-	justify-content: flex-start;
-
-	svg {
-		margin-right: $grid-unit-10;
-	}
-}
-
 @include break-small() {
 	.editor-post-preview {
 		display: none;
 	}
 
-	.editor-post-preview__dropdown {
+	.block-editor-post-preview__dropdown {
 		display: flex;
 	}
 }
```

Comment:
This component seem  to have a lot of custom styles, It looks very similar to "More Menus" or "Block Settings Menus"  so I wonder why it needs all these styles and can't just rely on the default UI components styles. Maybe our UI components styles are not well thought in this situation.

Reply:
My aim here was to preserve the current way it looks in `edit-post` completely. I think we could consider removing these, but I also thing it would be better to tackle that in a separate PR.